### ISSUE TITLE
Unknown Error (1); Check your Dev Console and ~/.wakatime.log for more info

### ISSUE THREAD

**Issue by Lewistrick** [wakatime/atom-wakatime#88]
I just installed Wakatime in Atom and entered my API key. In the status bar of Atom, it says 'Error' and on hover, it gives the information as shown in the title.
I'm using Atom 1.34.0 on Windows 10. I'm not sure what my Dev Console is, but C:/Users/my_name/.wakatime.log doesn't exist.

**Comment by Lewistrick** [comment]
I found the Dev Console. It gives me the following error:
```
[WakaTime] [WARN] [wakatime.coffee:527] Error: Command failed: pythonw C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\cli.py --file D:\Projects\Flask\userlogin2\forms.py --plugin atom-wakatime/7.0.7 --lineno 21 --config C:\Users\my_name\.wakatime.cfg --alternate-project userlogin
Traceback (most recent call last):
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\cli.py", line 24, in <module>
    import wakatime
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\__init__.py", line 17, in <module>
    from .main import execute
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\main.py", line 25, in <module>
    from .api import send_heartbeats
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\api.py", line 22, in <module>
    from .packages.requests.exceptions import RequestException
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\packages\requests\__init__.py", line 84, in <module>
    from urllib3.contrib import pyopenssl
  File "C:\Users\my_name\.atom\packages\wakatime\lib\wakatime-master\wakatime\packages\urllib3\contrib\pyopenssl.py", line 46, in <module>
    import OpenSSL.SSL
  File "C:\Anaconda\lib\site-packages\OpenSSL\__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
  File "C:\Anaconda\lib\site-packages\OpenSSL\SSL.py", line 124, in <module>
    SSL_ST_INIT = _lib.SSL_ST_INIT
AttributeError: module 'lib' has no attribute 'SSL_ST_INIT'

Logger._log @ C:\Users\my_name\.atom\packages\wakatime\lib\logger.coffee:54
```
How to fix this?

**Comment by alanhamlett** [comment]
What version of Anaconda python are you using?

**Comment by Lewistrick** [comment]
Python 3.5.2 :: Anaconda 4.2.0 (64-bit).

**Comment by gandarez** [comment]
It seems to be fixed by updating the library `pyopenssl`

**Comment by alanhamlett** [comment]
This might also be fixed with [v13.0.4 of wakatime](https://github.com/wakatime/wakatime/blob/master/HISTORY.rst#1304-2020-02-05).
